sdks/java/io/pulsar/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins { id 'org.apache.beam.module' }
+applyJavaNature(automaticModuleName: 'org.apache.beam.sdk.io.pulsar')
+
+def pulsarVersion = '2.8.0'
```

Comment:
Please use 2.8.2.

Reply:
Is there a reason to use 2.8.2 instead of 2.8.2? When I start the connector 2.8.0 was the latest release, thats why the version, but actually I need to change it to 2.9.0 since the ReadDoFn in beam use a function declare in 2.9.0 to get the latest message in the Topic